README.md

```diff
@@ -98,121 +98,144 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
-### 0.77 - December 2023 Update
+### 0.78 - January 2024 Update
 
-In this release, we focused on new features, stability and improvements.
+In this release, we focused on stability and improvements.
 
 **Highlights**
 
- - New utility: Command Not Found PowerShell 7.4 module - adds the ability to detect failed commands in PowerShell 7.4 and suggest a package to install using winget. Thanks [@carlos-zamora](https://github.com/carlos-zamora)!
- - Keyboard manager does not register low level hook if there are no remappings anymore.
- - Added support for QOI file type in Peek. Thanks [@pedrolamas](https://github.com/pedrolamas)!
- - Added support for loading 3rd-party plugins with additional dependencies in PowerToys Run. Thanks [@coreyH](https://github.com/CoreyHayward)!
+ - New localization languages released: Arabic (Saudi Arabia), Hebrew, Persian and Ukrainian. We are going to assume we have some bugs. We want to fix them and are open for community help here too.
+ - Many dependency updates, aiming for security and stability.
+ - Fixed commonly reported PowerToys Run startup crashes after an upgrade.
+ - New settings and GPO policies to help control behavior after an upgrade. Thanks [@htcfreek](https://github.com/htcfreek)!
 
 ### General
 
- - Bump WPF-UI package version to fix crashes related to theme changes. (This was a hotfix for 0.76)
- - Fixed typo in version change notification. Thanks [@PesBandi](https://github.com/PesBandi)!
- - Code improvements and fixed silenced warnings introduced by upgrade to .NET 8.
- - Update copyright year for 2024.
- - Added setting to disable warning notifications about detecting an application running as Administrator.
+ - Added Arabic (Saudi Arabia) translation.
+ - Added Hebrew translation.
+ - Added Persian translation.
+ - Added Ukrainian translation.
+ - Improved the file watcher used across many utilities to consume less resources. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 
 ### AlwaysOnTop
 
- - Show notification when elevated app is in the foreground but AlwaysOnTop is running non-elevated.
+ - Fixed an invisible border issue when the border color was set to the black color.
+ - Added the AlwayOnTop icon to the base application executable. Thanks [@ckirby19](https://github.com/ckirby19)!
 
 ### Command Not Found
 
- - Added a new utility: A Command Not Found PowerShell 7.4 module. It adds the ability to detect failed commands in PowerShell 7.4 and suggest a package to install using winget. Thanks [@carlos-zamora](https://github.com/carlos-zamora)!
+ - Signed the PowerShell scripts used by Command Not Found installation process.
 
-### Environment Variables
+### File Explorer add-ons
 
- - Fixed issue causing Environment Variables window not to appear as a foreground window.
+ - Fixed an issue causing SVG Thumbnail generation to hang when trying to preview SVG files at the same time.
```

Comment:
```suggestion
 - Fixed an issue that caused SVG Thumbnail generation to hang when trying to preview multiple SVG files at the same time.
```

Reply:
This is not correct. You can't preview multiple SVG files at the same time in the preview pane.

Reply:
Oh now I get it. Yeah then it is correct.